server_addon/max/server/settings/publishers.py

```diff
@@ -185,6 +189,11 @@ class PublishersModel(BaseSettingsModel):
         "optional": True,
         "active": False,
     },
+    "ValidateRenderPasses": {
+        "enabled": True,
+        "optional": True,
```

Comment:
Out of curiosity, ff this plug-in is optional - why do we need to check it in the first place? What does it solve by enforcing e.g. the scene name to be in the output filepaths?

Reply:
actually it should be set to optional = False instead of being optional validator.